src/test/resources/data/AutomatedSessionRemindersTest.json

```diff
@@ -22,7 +22,7 @@
       "googleId": "AutSessRem.instructor",
       "courseId": "AutSessRem.course",
       "name": "Test Ins for Aut Sessions Reminder",
-      "email": "${support.email}",
+      "email": "${test.instructor}@gmail.com",
```

Comment:
That's because of the way we do live testing. 
1. emails sent to the support email is supposed to come to my inbox. This test ensures that it indeed does happen. 
2. If I use the actual support email address for a different test account, (e.g. for test.instructor), then I get spammed too much during testing.